data/governments.txt

```diff
@@ -1203,6 +1258,51 @@ government "Quarg (Hai)"
 		ship "Quarg Skylark"
 		ship "Quarg Wardragon"
 
+government "Quarg (Incipias)"
+	swizzle 0
+	color .88 .77 0
+	"display name" "Quarg"
+
+	"player reputation" 1
+	"attitude toward"
+		"Quarg" 1
+		"Quarg (Hai)" 1
+		"Quarg (Kor Efret)" 1
+		"Quarg (Gegno)" 1
+		"Merchant" .01
```

Comment:
Why are these Hai, merchant, and pirate things here? If you never escort these Quarg, it doesn’t need to exist. If you will end up doing so, ignore this comment.

Reply:
removed them